lib/ezid/configuration.rb

```diff
@@ -44,7 +44,7 @@ def initialize
       @password         = ENV["EZID_PASSWORD"]
       @host             = ENV["EZID_HOST"] || HOST
       @port             = ENV["EZID_PORT"] || PORT
-      @use_ssl          = true if ENV["EZID_USE_SSL"] == true.to_s
+      @use_ssl          = true unless ENV["EZID_USE_SSL"].casecmp("no")
```

Comment:
This will raise an exception if `ENV["EZID_USE_SSL"].nil?`

Is there still a purpose for the option? If not, I would be inclined to remove it.